app/src/main/java/org/mozilla/fenix/settings/quicksettings/QuickSettingsController.kt

```diff
@@ -60,14 +61,19 @@ interface QuickSettingsController {
     fun handleAndroidPermissionGranted(feature: PhoneFeature)
 
     /**
-     * @see [TrackingProtectionInteractor.onTrackingProtectionToggled]
+     * @see [ProtectionsInteractor.onTrackingProtectionToggled]
```

Comment:
In the future, I would suggest just doing refactors in a standalone commit so that all of the renaming and refactoring don't even up jumble with the main changes.

Reply:
Will do, sorry about that!